### PR TITLE
Update calibration code to master of ctapipe

### PR DESCRIPTION
**Pull request by FrancaCassol**
This PR update the calibration code to be compatible to the master of ctapipe which now contains the calibration containers and Components

### CODE REVIEW

File: lstchain/calib/camera/r0.py [hunk 1]
```diff
@@ -30,6 +30,10 @@ class CameraR0Calibrator(Component):
         Set to None if no Tool to pass.
     kwargs
     """
+    offset = Int(
+        300,
+        help='Define the offset of the baseline'
+    ).tag(config=True)
```
Comment (rlopezcoto) [inline lstchain/calib/camera/r0.py]: this is the same offset as the one intended in `ctapipe_io_lst`, right?

File: lstchain/tools/camera_calibration_param.json [hunk 2]
```diff
@@ -34,11 +34,11 @@
        "r1_sample_start": 2,
       "r1_sample_end": 38
    },
-    "LocalPeakIntegrator":{
+    "LocalPeakWindowSum":{
      "window_shift": 5,
      "window_width":12
     },
```
Comment (rlopezcoto) [inline lstchain/tools/camera_calibration_param.json]: this values we may need to discuss/optimize. A window width of 12 corresponds to 12 ns, right? This is certainly too much for normal pulses, not sure if there is any requirement that needs to be fulfilled here, @moralejo, do you know?

File: lstchain/calib/camera/r0.py (hunk 1 above)
Comment (FrancaCassol) [reply lstchain/calib/camera/r0.py]: In principle yes, but this time the offset is set by Pawel's code and he use 300 as baseline.
Till now this is the active offset, for that the offset subtraction is done in this code for the moment

File: lstchain/tools/camera_calibration_param.json (hunk 2 above)
Comment (FrancaCassol) [reply lstchain/tools/camera_calibration_param.json]: @ruben, this committed code is not what we are going to use, it is just an example. 
This are configuration files and the values can be change as one want. In the data of last days I am using 9 ns, but as you say one should do a study before fixing it

**Comment by FrancaCassol** [comment]
@vuillaut, @ruben, I don't understand the error, could you perhaps help?

I would like to have the code committed so that we can install it in La Palma so that people can play with it. By the way, where should I install it?

**Comment by rlopezcoto** [comment]
Found! 
The problem is this [PR 1079](https://github.com/cta-observatory/ctapipe/pull/1079) in ctapipe. So I would wait until tomorrow to see if it is fixed there and then we merge, or we merge either way since it is not a problem of our repo.

**Comment by rlopezcoto** [comment]
I'll merge this since it is urgent. Let's hope the ctapipe bug will be solved soon.

**Pull request merged**